README.md

```diff
@@ -29,6 +29,15 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
 * [MTAR builder](https://www.npmjs.com/package/mbt) (`npm install -g mbt`)
 * [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html), Install cf-cli and run command `cf install-plugin multiapps`.
 
+> [!Note]
+> The behaviour of clicking attachment and opening it varies based on the version of cds-services used by the CAP application. For cds-services version >= 3.4.0, clicking on attachment will
```

Comment:
Add the second line as bullet point. For example,

- For version >= 3.4.0
- For version < 3.4.0

Also after note, add a heading "cds-services"